internal/transport/http.go

```diff
@@ -125,7 +128,10 @@ func (t *HTTP) Connect() error {
 					}
 					_ = t.dataHandler("data", metadata, data)
 				}
-				resp.Body.Close()
+				err = resp.Body.Close()
+				if err != nil {
```

Comment:
```suggestion
				if err := resp.Body.Close(); err != nil {
```

Reply:
No, I do not want to mix fixing issue and doing refactoring of code.